pkg/virt-config/config-map.go

```diff
@@ -150,6 +154,8 @@ func defaultClusterConfig() *Config {
 		EmulatedMachines:       emulatedMachinesDefault,
 		LessPVCSpaceToleration: DefaultLessPVCSpaceToleration,
 		NodeSelectors:          nodeSelectorsDefault,
+		NetworkInterface:       defaultNetworkInterface,
+		DisableSlirpInterface:  true,
```

Comment:
Why do we disable it by default? It was not disabled before.

Reply:
Mostly from security reasons. It's not supported by enterprise distributions. After speaking with @fabian the agreement was that it's better to disable it by default and make it opt-in instead.